### PR TITLE
Increase coverage chsdi.views.stationboard

### PR DESCRIPTION
**Pull request by mariavkourouni**
now it is 100%

### CODE REVIEW

File: chsdi/tests/integration/test_stationboard.py [hunk 1]
```diff
@@ -10,6 +10,11 @@ def test_stationboard(self):
         resp = self.testapp.get('/stationboard/stops/8501120', params=params, status=200)
         self.assertEqual(resp.content_type, 'application/json')
 
+    def test_stationboard_nodata(self):
+        params = {'destination': 'myDestination'}
+        resp = self.testapp.get('/stationboard/stops/8501120', params=params, status=200)
```
Comment (loicgasser) [inline chsdi/tests/integration/test_stationboard.py]: can you test that len(results) is equal to 0?

Comment (loicgasser) [inline chsdi/tests/integration/test_stationboard.py]: See: https://github.com/geoadmin/mf-chsdi3/blob/chsdi.views.stationboard.py/chsdi/views/stationboard.py#L73

File: chsdi/tests/integration/test_stationboard.py [hunk 2]
```diff
@@ -42,3 +47,7 @@ def test_available_destinations_with_cb(self):
         params = {'callback': 'cb'}
         resp = self.testapp.get('/stationboard/stops/8500109/destinations', params=params, status=200)
         self.assertEqual(resp.content_type, 'application/javascript')
+
+    def test_stationboard_destination_nodata(self):
+        resp = self.testapp.get('/stationboard/stops/123/destinations', status=200)
```
Comment (loicgasser) [inline chsdi/tests/integration/test_stationboard.py]: same here, look for no data json response.

**Comment by loicgasser** [comment]
2 small comments otherswise it's good. thanks!

**Comment by mariavkourouni** [comment]
Yes, now it makes sense @loicgasser  
Thank you!

**Comment by loicgasser** [comment]
Great thanks!

**Pull request merged**